pkg/logutil/logutil.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package logutil
+
+import (
+	logging "gopkg.in/op/go-logging.v1"
+
+	"k8s.io/klog/v2"
+)
+
+func WrapGoLogging() {
+	logging.SetBackend(klogGoLoggingBackend{})
+}
+
+type klogGoLoggingBackend struct{}
+
+func (k klogGoLoggingBackend) Log(level logging.Level, calldepth int, rec *logging.Record) error {
+	switch level {
+	case logging.CRITICAL:
+		klog.ErrorDepth(calldepth+1, rec.Message())
```

Comment:
Anyone know how to set both verbosity level and call depth?
It's not clear how some of these go-logger levels should be mapped to klog levels.

Reply:
figured it out and updated the lib to support configurable mapping